### PR TITLE
iOS-1591 Copy settings from the active view when create new

### PR DESCRIPTION
**Pull request by joe-pusya**

### CODE REVIEW

File: Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift [hunk 1]
```diff
@@ -99,11 +99,10 @@ final class SetViewTypesPickerViewModel: ObservableObject {
     
     private func createView() {
         let name = name.isEmpty ? Loc.SetViewTypesPicker.Settings.Textfield.Placeholder.untitled : name
-        Task {
-            try await dataviewService.createView(
-                DataviewView.created(with: name, type: selectedType),
-                source: source
-            )
+        Task { [weak self] in
+            guard let self else { return }
```
Comment (mgolovko) [inline Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift:103]: you capture self immediately when task started for all async calls

**Review by mgolovko — APPROVED** [review]

File: Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift (hunk 1 above)
Comment (joe-pusya) [reply Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift:103]: you mean we don't nee to make this `guard let self else { return }`?

Comment (mgolovko) [reply Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift:103]: yep

Comment (joe-pusya) [reply Anytype/Sources/PresentationLayer/TextEditor/Set/Views/Popups/ViewTypes/SetViewTypesPickerViewModel.swift]: Removed in this case

**Pull request merged**